os_stub/spdm_crypt_ext_lib/cryptlib_ext.h

```diff
@@ -426,24 +426,6 @@ extern bool libspdm_hkdf_sm3_256_extract_and_expand(const uint8_t *key, size_t k
 extern bool libspdm_ec_set_pub_key(void *ec_context, const uint8_t *public_key,
                                    size_t public_key_size);
 
-/**
```

Comment:
Please restore this as part of https://github.com/DMTF/libspdm/pull/1845/files#r1153815678